src/test/regress/expected/multi_complex_expressions.out

```diff
@@ -347,3 +347,155 @@ SELECT count(*) FROM lineitem JOIN orders ON l_orderkey = o_orderkey
 SELECT count(*) FROM lineitem, orders WHERE l_orderkey + 1 = o_orderkey;
 ERROR:  cannot perform local joins that involve expressions
 DETAIL:  local joins can be performed between columns only
+-- Check that we can issue limit/offset queries
+-- SIMPLE LIMIT/OFFSET with ORDER BY
+SELECT o_orderkey FROM orders ORDER BY o_orderkey LIMIT 10 OFFSET 20;
+ o_orderkey 
+------------
+         69
+         70
+         71
+         96
+         97
+         98
+         99
+        100
+        101
+        102
+(10 rows)
+
+-- LIMIT/OFFSET with a Subquery
+SET citus.task_executor_type TO 'task-tracker';
+SELECT 
+	t.o_custkey,
+	COUNT(*) AS ss
+FROM 
+	(SELECT o_custkey, COUNT(*) AS order_count FROM orders GROUP BY o_custkey) t
+GROUP BY 
+	t.o_custkey
+ORDER BY 
+	t.o_custkey DESC
+LIMIT 10 OFFSET 20;
+ o_custkey | ss 
+-----------+----
+      1466 |  1
+      1465 |  1
+      1463 |  1
+      1462 |  1
+      1460 |  1
+      1459 |  1
+      1457 |  1
+      1456 |  1
+      1454 |  1
+      1453 |  1
+(10 rows)
+
+SET citus.task_executor_type TO 'real-time';
+-- Ensure that we push down LIMIT and OFFSET properly
+EXPLAIN (COSTS FALSE) SELECT o_custkey FROM orders LIMIT 10 OFFSET 15;
+                        QUERY PLAN                         
+-----------------------------------------------------------
+ Distributed Query into pg_merge_job_420038
+   Executor: Real-Time
+   Task Count: 2
+   Tasks Shown: One of 2
+   ->  Task
+         Node: host=localhost port=57637 dbname=regression
+         ->  Limit
+               ->  Seq Scan on orders_290006 orders
+ Master Query
+   ->  Limit
+         ->  Seq Scan on pg_merge_job_420038
+(11 rows)
+
+EXPLAIN (COSTS FALSE) SELECT o_custkey FROM orders GROUP BY o_custkey LIMIT 10 OFFSET 15;
```

Comment:
Ohh, with the introduction of COSTS FALSE, we can't see the row numbers pulled from worker nodes. We can remove these tests.